### ISSUE TITLE
*.swiftypecdn.com and *.swiftype.com

### ISSUE THREAD

**Issue by terrorist96** [EFForg/privacybadger#1562]
https://www.bia2.com/search/#stq=test

`s.swiftypecdn.com` and `api.swiftype.com` break search results.
`search-api.swiftype.com` breaks the recent results on right.

![image](https://user-images.githubusercontent.com/12802056/29010739-736b4748-7afb-11e7-99cf-a8d32a268c46.png)

**Comment by ghostwords** [comment]
More examples:

- Search box at the top of `http://www.oldtownhotsprings.org/`
- `https://shop.sportsbasement.com/search?query=running+shorts`
- Search widget at the bottom of `http://docs.ansible.com/ansible/latest/ec2_facts_module.html`

**Comment by ghostwords** [comment]
I can't figure out how to get `swiftypecdn.com` to show up as tracking.

**Comment by terrorist96** [comment]
So swiftypecdn.com won't be fixed? My PB learned to block it, so I'm sure others' installation did too and are affected by this.

**Comment by ghostwords** [comment]
Can you reproduce `swiftypecdn.com` getting blocked with a fresh profile?

**Comment by terrorist96** [comment]
No. But then why did it learn it to begin with? Do we need another migration to fix incorrectly learned rules?

**Comment by cowlicks** [comment]
@terrorist96 Yes, usually.
